src/python/pants/backend/python/subsystems/pex_bootstrap/BUILD

```diff
@@ -0,0 +1,7 @@
+# Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+python_library(
+  name='ipex_launcher',
```

Comment:
(1) It is intended to be a reference to [Bazel's ijar tool](https://github.com/bazelbuild/bazel/tree/master/third_party/ijar), which produces smaller "interface" jars without any method implementations. I don't think python necessarily has the same concept. They are both "a smaller version of some zip file, which is useful for build tools, but cannot be executed on its own".

I will document what ipex stands for in the option help!

(2) Ok, will do! Will remove references to "pex_bootstrap", rename the target and directory to "ipex".

Reply:
> (1) It is intended to be a reference to [Bazel's ijar tool](https://github.com/bazelbuild/bazel/tree/master/third_party/ijar), which produces smaller "interface" jars without any method implementations. I don't think python necessarily has the same concept. They are both "a smaller version of some zip file, which is useful for build tools, but cannot be executed on its own".
> 
> I will document what ipex stands for in the option help!

I'm pretty sure I coined ipex meaning "incremental pex" (as in, a pex file which isn't complete, but has work still to do before it can run)... I'm open to better names :)

Reply:
Aha, ok - thanks for that background @illicitonion. The `i` at least makes sense now.

@cosmicexplorer I'll continue to hold off review until green CI.

Reply:
@jsirois that's great. I'll ping you and other reviewers on green CI.